src/utils/use-swr-config.ts

```diff
@@ -5,7 +5,7 @@ import { mergeObjects } from './helper'
 import { FullConfiguration, Cache } from '../types'
 
 export const useSWRConfig = <
-  T extends Cache = Map<string, any>
+  T extends Cache = Cache
```

Comment:
Should this use `Cache<Map<string, any>>` instead? I didn't think that was necessary, but I'm not sure.

Reply:
I guess this will be a breaking change as well (for those supplying a custom type in `useSWRConfig`)? Maybe we should update release notes?

Reply:
We added it recently only for v2, so I think it's fine

Reply:
Apparently this change was added recently — not sure why it was using `Map<string, any>` https://github.com/vercel/swr/pull/1938

Reply:
The previous cache type in v1 is `Map<string, any>`, it's good to update it now 👍